tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html

```diff
@@ -892,6 +933,29 @@
   _equals: function(a, b) {
     return a === b;
   },
+  _getCurrentDeviceParams: function(colorByParams) {
+    var deviceParams = _.filter(colorByParams.device, function(param) {
+      return _.some(DEVICE_NAMES_INCLUDE, function(rule) {
+        return rule.regex.test(param.device);
+      });
+    });
+    // Remove common prefix and merge back corresponding color. If
```

Comment:
For my edification, could you please clarify why the common prefix is not useful? Are there cases in which the prefixes would differ? Thanks!

Reply:
Ah, I see. We compute the common prefix. Thank you!